api/swagger.yaml

```diff
@@ -55,8 +55,8 @@ info:
     the URL is not supported by the daemon, a HTTP `400 Bad Request` error message
     is returned.
 
-    If you omit the version-prefix, the current version of the API (v1.44) is used.
-    For example, calling `/info` is the same as calling `/v1.44/info`. Using the
+    If you omit the version-prefix, the current version of the API (v1.45) is used.
+    For example, calling `/info` is the same as calling `/v1.45/info`. Using the
```

Comment:
FWIW; there was discussion to continue deprecating the "version-less" API, and to pin this to a fixed version, but that probably requires more changes, so let's keep that separate for now